envi/archs/i386/emu.py

```diff
@@ -761,6 +761,19 @@ def i_div(self, op):
             self.setRegister(REG_EAX, quot)
             self.setRegister(REG_EDX, rem)
 
+        elif oper.tsize == 8:
```

Comment:
Totally!  Completely agreed.  I was just saying we could move the REG_FOO constants to somewhere to isolate the circular dependency. :)